packages/backend/src/shared/utils.test.ts

```diff
@@ -251,6 +248,13 @@ describe('utils', (): void => {
       )
       ctx.request.body = requestBody
 
+      await expect(
+        verifyApiSignature(ctx, {
+          ...Config,
+          adminApiSecret: 'test-secret'
+        })
+      ).resolves.toBe(true)
```

Comment:
make the test agnostic to the actual implementation of cache (just call the signature verification twice directly)